database/seeders/IdentitySeeder.php

```diff
@@ -18,15 +18,24 @@ public function run()
             $environment = config('app.env');
         }
 
-        if (Storage::disk('seeds')->exists(sprintf('identities.%s.json', $environment))) {
+        // fix for when it runs in environments without access to S3 bucket
+        try {
+            // try connecting to the seeds S3 bucket
+            Storage::disk('seeds');
+        } catch (\Exception $e) {
+            // mock the seeds filesystem locally
+            Storage::fake('seeds');
+        }
```

Comment:
As an alternative to this, why not have a local and s3 version of the seed disk (e.g. seed-local, seed-s3) and change between them based on an environment variable. For example have an ENV like `SEED_DRIVER` and in the example env set that to `seed-local` but in the deployment configuration set that to `seed-s3`. In that way you won't have to do all the exception checking in each seeder and you can run the backups locally; for example to test the backup system.

See: https://laracasts.com/discuss/channels/laravel/how-to-use-local-filesystem-for-development-but-s3-for-production